piet-direct2d/examples/basic.rs

```diff
@@ -101,3 +106,124 @@ fn main() {
     )
     .unwrap();
 }
+
```

Comment:
It looks like a lot of this now exists in d3d.rs, so it's duplication. I'm not sure it's worth blocking the PR to resolve that, but maybe a TODO, and this can be followup for an interested person.

Reply:
added a comment to the example.